.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: licensing
+  name: Add license headers to source files
+  description: This script adds license headers to source files in the repository
+  entry: license-files
```

Comment:
Will this turn into `license_files`? I'm not sure the relationship between this and the file `mbed_tools_ci_scripts/license_files.py`. Maybe there is something executable in the path with the name `license-files`?

Reply:
`license-files` the `setuptools` entry point as listed in `setup.py` https://github.com/ARMmbed/mbed-tools-ci-scripts/blob/master/setup.py#L48